crates/libtest2/src/macros.rs

```diff
@@ -53,11 +79,8 @@ macro_rules! _test_parse {
                 fn run $($item)*
 
                 $(
-                    match $crate::_private::parse_ignore!($($attr)*) {
-                        ::std::option::Option::None => context.ignore()?,
-                        ::std::option::Option::Some(reason) => context.ignore_for(reason)?,
-                    }
-                )*
+                    $crate::_private::parse_ignore!(context, $ignore)?;
+                )?
```

Comment:
Should we move the conditional nature of `$ignore` into `_parse_ignore`, simplifying this?